### ISSUE TITLE
Build issues

### ISSUE THREAD

**Issue by ghost** [9seconds/rymtracks#18]
During the course of trying to pack the RYMTracks was trying to downloading a distribute-0.6.21 but this is already merged to pysetuptools.

From the administrator of the http://slackbuilds.org/ :

Here's the problem i found

Downloading
http://pypi.python.org/packages/source/d/distribute/distribute-0.6.21.tar.gz
Extracting in /tmp/easy_install-3wvy2u/nltk-2.0.4/temp/tmpk1p28X
Now working in
/tmp/easy_install-3wvy2u/nltk-2.0.4/temp/tmpk1p28X/distribute-0.6.21
Building a Distribute egg in /tmp/easy_install-3wvy2u/nltk-2.0.4
/tmp/easy_install-3wvy2u/nltk-2.0.4/distribute-0.6.21-py2.7.egg

downloading something from the SlackBuild during the build process is
something we tried to avoid, since the user might not aware of this
(this is how i missed it in the initial submission)

distribute is already merged to pysetuptools
perhaps you can talk to the author to have a look on this issue?

thanks

**Comment by 9seconds** [comment]
Hi,

Sure I will. But now I do not sure that it make sense to break their pipeline. As far as I understand, you are maintaining RYMTracks package for Slakware, right? I had no deal with your packaging system (I did only RPMs and DEBs) but I guess it is not correct from packaging perspective to allow user to install anyting from PyPI during the process. I freezed the version numbers of dependencies but they still have those horrible ">=" which means that it might be absolutely possible to have working and broken installation during absolutely the same process but within a different time.

Anyway right now I am going to make capitalization support optional because it looks like that setuptools (which is awful mess) and pip giving some new behavior, it is really hard to force them all to download NLTK data on install. It means that step you mentioned will be postponed.
